cli/src/commands/logs.ts

```diff
@@ -6,13 +6,21 @@ import {
 } from '../utils/helpers';
 import ms = require('ms');
 
+const detailedLogsRegexps = [
+  /Function invocation took [\d.]+ us/,
```

Comment:
Shouldn't this be `/^Function ...` (or perhaps the pattern is so rare than it's not interesting?)

Reply:
You're right, I didn't anchor it.